homeassistant/components/mochad.py

```diff
@@ -64,15 +83,26 @@ def start_mochad(event):
 class MochadCtrl:
     """Mochad controller."""
 
-    def __init__(self, host, port):
-        """Initialize a PyMochad controller."""
+    def __init__(self, host, port, broker, mqtt_port, username, password,
```

Comment:
Please start the signature with `hass` parameter.

Reply:
Moved up as suggested